sdk-core/src/main/java/care/data4life/sdk/attachment/AttachmentService.kt

```diff
@@ -125,8 +126,10 @@ class AttachmentService internal constructor(
                     originalData,
                     if (position == POSITION_PREVIEW) DEFAULT_PREVIEW_SIZE_PX else DEFAULT_THUMBNAIL_SIZE_PX
                 )
-                if (downscaledId != null)
-                    additionalIds.add(downscaledId)
+
+                if (downscaledId == null) return null
+
+                additionalIds.add(downscaledId)
```

Comment:
Please note: This is the mentioned rollback. See also [here](https://github.com/d4l-data4life/hc-sdk-kmp/blob/aab1e4a026242912c387f1557f60593393503983/sdk-core/src/main/java/care/data4life/sdk/AttachmentService.java#L83-L85) .
However this only a theoretical bug, since if no JEPGWriter is once available it should not be if it try it a 2nd time (except there is a super weird platform in place)